docs/motivation.md

```diff
@@ -0,0 +1,550 @@
+# The Verona language: Core type system
+
+Verona is a language with a behaviour-oriented concurrency model and a region abstraction for ownership.
+These are fundamentally new abstractions in language design and are covered separately.
+This document aims to cover the parts of the type system that are similar to (though in some places diverge in interesting ways from) other languages.
+
+Verona is an object oriented reference capability language with structural
+subtyping and bounded polymorphism. This document aims to describe the aspects
+of the type system that are independent from both the concurrency model and
+reference capabilities.
+
+## Structural type system
+
+A type system is all about managing a complexity budget. Make it too complicated
+and programmers will find it difficult to use and be less productive. Make it too
+simple and it will give you less guarantees to lean on as a programmer.
```

Comment:
Similarly, we're provoking argument with an implicit productivity claim. For this to be effective, we need to back up the claim. Also s/less/fewer/.

Reply:
Evidence on productivity?  Jan Vitek paper about productivity citation.  We approximate dynamic programming through the really cool typesystem.